aries-site/src/pages/templates/global-banner-notifications.mdx

```diff
@@ -0,0 +1,161 @@
+import { Anchor } from 'grommet';
+import {  BestPracticeGroup, Example } from '../../layouts';
+import {
+  BannerContentLayoutExample,
+  BannerNotificationCritical,
+  BannerNotificationCriticalClose,
+  BannerNotificationDoExample,
+  BannerNotificationDontExample,
+  BannerNotificationInfo,
+  BannerNotificationWarning,
+  BannerNotificationWarningClose,
+  BannerNotificationDiagram,
+} from '../../examples';
+
+
+<Example
+  showResponsiveControls={['fullScreen']}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  caption="An example of a dashboard with a Global Banner warning the user that their subscription is expiring. The user can either dismiss the notification or click on the anchor to renew their subscription."
+  screenContainer={{ scale: 0.57 }}
+>
+  <BannerContentLayoutExample />
+</Example>
+
+
+## Guidance
+Global Banners are high-attention notifications used primarily for status updates on background processes and upcoming system changes and releases.
+
+### Usage
+- Global Banners are used to showcase Warning and Critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
+- Since Global Banners take over the top of an interface and can be intrusive, they should be reserved only for important system generated updates.
+- Also when applicable, Global Banners should always have a relevant anchor(s) to direct users to further actions or to view more about the issue in full detail.
+
+### Do Not Use Global Banners For
+- Completion of tasks, such as long running or batch actions.
+- Success or failure confirmations.
+- Low-attention and low-severity information.
+  - For low-severity notifications that do not require user action or user attention such as confirmation receipts, instead use [Toast Notifications](templates/toast-notifications).
+
+### Things to Avoid
+- Do not use use close icon when information is important and needs to remain persistent. Also consider whether or not the information should remain persistent across multiple sessions
```

Comment:
```suggestion
- Do not use `onClose` when information is important and needs to remain persistent. Also, consider whether or not the information should remain persistent across multiple sessions.
```
also, I don't understand the second sentence. are we saying that if information needs to remain persistent across multiple sessions it should never be dismissible? or that we should persist a banner until a user dismisses it (meaning it could persist over multiple sessions but they can dismiss it anytime, afterwhich it will not appear?) @KennyAtHPE

Reply:
It'd be the latter, maybe something along the lines of "Also, global banners may remain persistent across multiple sessions until they are dismissed." would be more clear.

Reply:
Cool then I think we should move that sentence from the "Things to Avoid" and instead just place it under guidance. @britt6612